R/tmb.R

```diff
@@ -154,12 +154,14 @@ h_mmrm_tmb_formula_parts <- function(formula) {
       stats::as.formula(paste(". ~ . +", cov_vars$group_var))
     )
   }
+  cov_type <- deparse(cov_term[[1L]])
   structure(
-    list(
+    .Data = list(
       formula = formula,
       model_formula = model_formula,
       full_formula = full_formula,
-      cov_type = deparse(cov_term[[1L]]),
+      cov_type = cov_type,
+      spatial = cov_type %in% cov_type_spatial,
```

Comment:
```suggestion
      is_spatial = cov_type %in% cov_type_spatial,
```
and adapt downstream